WordPress/Resources/zh-Hans.lproj/Localizable.strings

```diff
@@ -7768,7 +7826,7 @@ translators: %s: Select control option value e.g: \"Auto, 25%\". */
 "To use stats on your site, you'll need to install the Jetpack plugin." = "要在您的网站上使用“统计”功能，您需要安装Jetpack插件。";
 
 /* Message explaining that Jetpack needs to be installed for a particular site. Reads like 'To use this app for example.com you'll need to have... */
-"To use this app for %@ you'll need to have the Jetpack plugin installed and activated." = "要将此应用用于 ％@，您需要安装并启用 Jetpack 插件。";
+"To use this app for %@ you'll need to have the Jetpack plugin installed and activated." = "要将此应用用于 %@，您需要安装并启用 Jetpack 插件。";
```

Comment:
The previous version had the wrong `%` character which wouldn't have worked in `String(format:)`.

Reply:
Properly flagged by the linter, I presume?